### ISSUE TITLE
Not possible to disable indicators in status bar without also disabling activity bar for scan per file

### ISSUE THREAD

**Issue by metablaster** [Gruntfuggly/todo-tree#703]
Here is my test screenshot to show what I mean:

![alt](https://i.imgur.com/hqQ218P.png)

I want to set up `todo-tree` so that indicators marked in green circle (in screenshot activity bar) stay but indicators marked in red on status bar should go away completely.

My current relevant settings are this:

```json
// I want to see indicators in activity bar (green circle in screenshot) only for current file
"todo-tree.general.statusBar": "current file",

"todo-tree.general.showIconsInsteadOfTagsInStatusBar": true,

// must be enabled to see indicator in activity bar (green circle in screenshot)
"todo-tree.general.showActivityBarBadge": true,

"todo-tree.tree.scanMode": "workspace",
```

What combination of settings I should use and configure to achieve this?
I've tried I believe everything but nothing works.

```
Todo Tree v0.0.220

Version: 1.73.1 (system setup)
Commit: 6261075646f055b99068d3688932416f2346dd3b
Date: 2022-11-09T04:27:29.066Z
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19045
Sandboxed: No
```

If I set this:

```json
"todo-tree.general.statusBar": "none"
```

What happens is that there are no icons in status bar as desired, however activity bar is now spammed:

![](https://i.imgur.com/6mSgxGI.png)

Indicator count in the activity bar is for entire repo but I would like to see it only for current file or opened files without seeing anything on status bar.

is that possible?

Btw. this settings also does not help
```json
"todo-tree.tree.scanMode": "curent file"
```

**Comment by Gruntfuggly** [comment]
Not possible with the current settings - will try and add it when I get some spare time.

**Comment by metablaster** [comment]
Thank you, I'm looking forward for this feature, I'm really a big fan of todo-tree, certainly the best for the purpose.

**Comment by Gruntfuggly** [comment]
Should be fixed in the latest version (0.0.221) which will be released soon.

**Comment by metablaster** [comment]
Thank you for implementing!
I'll test it when it get's updated.
